### PR TITLE
[WIP] Potential fix to fixed-position issue

### PR DESCRIPTION
**Pull request by jridgewell**

### CODE REVIEW

**Comment by jridgewell** [comment]
Whooo, I managed to detect stickies with `position: auto` vs `position: 0px`! @dvoytenko

**Comment by erwinmombay** [comment]
ping @dvoytenko for review. would like to expedite this so we can patch last weeks canary and move it into 1%

File: src/service/fixed-layer.js [hunk 1]
```diff
@@ -269,9 +269,14 @@ export class FixedLayer {
...
         // 1. Set all style top to `auto` and calculate the auto-offset.
         for (let i = 0; i < elements.length; i++) {
-          const fe = elements[i];
-          oldTops.push(getStyle(fe.element, 'top'));
-          setStyle(fe.element, 'top', 'auto');
+          const el = elements[i].element;
+          const style = computedStyle(this.ampdoc.win, el);
+          styles.push(style);
+          if (endsWith(style.position || '', 'sticky')) {
+            setStyle(el, 'position', 'fixed');
```
Comment (dvoytenko) [inline src/service/fixed-layer.js]: What does this attempt to do? Generally, `position: sticky` should be completely ignored for transfer layer. It's simply impossible to transfer it.

Comment (jridgewell) [inline src/service/fixed-layer.js]: This isn't for transfering, but to detect `top: auto` stickies. In non-safari, we still need to add the `top: ${padding}`, which we cannot do if the sticky only has a `bottom`

**Pull request merged**